relays/headers-relay/src/sync_loop_tests.rs

```diff
@@ -18,30 +18,30 @@
 
 use crate::sync_loop::{run, SourceClient, TargetClient};
 use crate::sync_types::{HeadersSyncPipeline, QueuedHeader, SourceHeader, SubmittedHeaders};
-use crate::utils::{process_future_result, retry_backoff, HeaderId, MaybeConnectionError};
 
 use async_trait::async_trait;
 use backoff::backoff::Backoff;
 use futures::{future::FutureExt, stream::StreamExt};
 use parking_lot::Mutex;
+use relay_utils::{process_future_result, retry_backoff, HeaderId, MaybeConnectionError};
 use std::{
 	collections::{HashMap, HashSet},
 	sync::Arc,
 	time::Duration,
 };
 
-type TestNumber = u64;
-type TestHash = u64;
-type TestExtra = u64;
-type TestCompletion = u64;
-type TestHeaderId = HeaderId<TestHash, TestNumber>;
-type TestQueuedHeader = QueuedHeader<TestHeadersSyncPipeline>;
+pub type TestNumber = u64;
```

Comment:
It's the test code, but maybe still worth adding docs for at least some of the stuff?